docs/learn/distributed-authority-quick-start.md

```diff
@@ -11,12 +11,14 @@ Before you begin, you need the following:
 
 - An active Unity account with a valid license.
 - The [Unity Hub](https://unity.com/download).
-- A supported version of the Unity Editor. Refer to the [Netcode for GameObjects requirements](https://docs-multiplayer.unity3d.com/netcode/current/installation).
+- A supported version of the Unity 6 Editor. 
+  - Additional requirements information can be found here: [Netcode for GameObjects requirements](https://docs-multiplayer.unity3d.com/netcode/current/installation).
 
 ## Install packages
```

Comment:
I believe this section should move to below project setup.

Reply:
Doesn't a user need to know what editor to use prior to creating a project?